app/controllers/application_controller.rb

```diff
@@ -246,4 +246,14 @@ def configure_permitted_parameters
       user_params.permit(:username, :email, :password, :password_confirmation)
     end
   end
+
+  def find_class(asset)
+    classes = ActiveRecord::Base.connection.tables.map { |f| f.classify }
+    find = classes.find { |m| m == asset.classify }
+    if find
+      find.safe_constantize
+    else
+      raise "Unknown resourse"
+    end
+  end
```

Comment:
I think that more effective way will be
class_name = ActiveRecord::Base.connection.tables.find{|t| t == asset}.try(:classify)
class_name ? class_name.safe_constantize : raise "Unknown resourse"